src/omniperf_soc/soc_gfx942.py

```diff
@@ -69,6 +69,11 @@ def __init__(self, args, mspec):
         )
         # self.roofline_obj = Roofline(args, self._mspec)
 
+        # --showmclkrange is broken in MI308X, hardcode freq
```

Comment:
The rocm-smi --showmclkrange fix is already in mainline. So we can exclude this work-around.

Reply:
Is it in a ROCm release? More specifically: Is it in the _specific_ minumum ROCm version we expect this customer to use?

If not, we need to either hard-code or test whether it works, and use the hard-coded values as a fall-back.

Reply:
Good point. Looks like it will be in 6.2.

We should keep this work-around in, then.